### PR TITLE
[TR] PIM-4144 Add fields for reference data

### PR DESCRIPTION
**Pull request by filipsalpe**

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Normalizer/ReferenceDataConfigurationNormalizer.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+     */
+    public function normalize($config, $format = null, array $context = [])
+    {
+        return [
+            'name'  => $config->getName(),
+            'type'  => $config->getType(),
+            'class' => $config->getClass()
+        ];
+    }
+
```
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Normalizer/ReferenceDataConfigurationNormalizer.php]: `$config` should be type hinted as it clearly require an interface

Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Normalizer/ReferenceDataConfigurationNormalizer.php]: We should check if $config implements  `ConfigurationRegistryInterface`

Comment (filipsalpe) [inline src/Pim/Bundle/EnrichBundle/Normalizer/ReferenceDataConfigurationNormalizer.php]: supportsNormalization?

**Comment by sumbobyboys** [comment]
GTM after comment fix

File: src/Pim/Bundle/EnrichBundle/Normalizer/ReferenceDataConfigurationNormalizer.php (hunk 1 above)
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Normalizer/ReferenceDataConfigurationNormalizer.php]: GTM

**Pull request merged**